engines/qemu/test-image/upload.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+# Find location of the script no matter where it's located
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+echo "Skipping tinycore-setup.tar.lz4 as we don't change it"
+# aws s3 cp "$DIR/tinycore-setup.tar.lz4" "s3://public-qemu-images/test-images/tinycore-setup.tar.lz4"
```

Comment:
Only, need to update it if we change it... And changes are manual, so we won't do that very often... I'll add additional comment..